### ISSUE TITLE
Hello Triangle crashes Visual Studio Graphics Debugger

### ISSUE THREAD

**Issue by mcleary** [microsoft/DirectX-Graphics-Samples#601]
I am trying to use the Visual Studio Graphics Debugger in order to understand more about how DX12 works under the hood but the Hello Triangle crashes it when I try to capture a frame.

The following is a screenshot of my Visual Studio About dialog:
![image](https://user-images.githubusercontent.com/1537712/69101567-d7f7df80-0a57-11ea-9cbe-ef8d585cc879.png)

As you can see I also have the optional *Graphic Tools* installed:
![image](https://user-images.githubusercontent.com/1537712/69101619-fcec5280-0a57-11ea-9054-ecaf5da1fe3a.png)

In order to reproduce the crash, just launch the Hello Triangle sample with **ALT+F5**. When the application launches, hit **Print Screen** or click the **Capture Frame** button. When I do this I get the following in Visual Studio:

![image](https://user-images.githubusercontent.com/1537712/69101796-784e0400-0a58-11ea-89ba-464afbd729c7.png)

And the callstack is massive but most calls are in this **DXCaptureReplay.dll**.

In summary, my final question is whether this is a problem with the sample or with the debugger and where is the best place to report this.

Thanks

**Comment by weltkante** [comment]
From what I understand the VS Graphics Debugger is no longer actively supported, it looks like VS GD can only be used up to Windows SDK 14393 which is pretty old by now: #586

> The problem is you use never version of API (SDK above 14393) and it is not supported by GD, use pix instead

**Comment by mcleary** [comment]
I see, didn't know the graphics debugger is no longer supported. Thanks for the info. I will try pix.
